crates/cxx-qt-lib/src/core/qmodelindex.rs

```diff
@@ -35,6 +36,9 @@ mod ffi {
         #[rust_name = "sibling_at_row"]
         fn siblingAtRow(self: &QModelIndex, row: i32) -> QModelIndex;
 
+        /// Returns a `quintptr` used by the model to associate the index with the internal data structure.
+        #[rust_name = "internal_id"]
+        fn internalId(self: &QModelIndex) -> quintptr;
```

Comment:
Note that this is a breaking change.
However, let's do this now while we are still pre-1.0

Reply:
Of course it's the same fo all other APIs that were changed during this PR.